doc/uftrace-info.md

```diff
@@ -4,33 +4,40 @@
 
 NAME
 ====
-uftrace-info - Print tracing information for trace data
+uftrace-info - Print tracing information from the current trace data file
 
 SYNOPSIS
 ========
-uftrace info [*options*] [*COMMAND*]
+uftrace info [*options*]
 
 DESCRIPTION
 ===========
-This command prints metadata recorded in the header of a given data file.
+This command prints metadata recorded in the header of a given trace data file.
+
+Alternatively, it can:
+
+- be used to generate a symbol table of the traced symbols
+
+- print a task relationship tree for the recorded tasks in the trace
 
 
 OPTIONS
 =======
-\--symbols
-:   Print symbols table instead of the recorded tracing info.  It will print
-    two symbol tables - normal symbols and dynamic symbols.  The normal symbols
-    are from the executable itself, and dynamic symbols are for library calls.
-    When COMMAND is given, it should provide symbol information which might not
-    be available from the recorded path of 'exe image' or the symbol file in the
-    data directory.
+\--symbols [*EXECUTABLE*]
```

Comment:
I forgot why it needs the EXECUTABLE argument or COMMAND, maybe we can omit it now.